clang/lib/CodeGen/CGCall.cpp

```diff
@@ -199,15 +199,17 @@ static void appendParameterTypes(const CodeGenTypes &CGT,
                               prefix.size());
 }
 
+using SmallExtParameterInfoList =
+    SmallVector<FunctionProtoType::ExtParameterInfo, 16>;
+
 /// Arrange the LLVM function layout for a value of the given function
 /// type, on top of any implicit parameters already stored.
 static const CGFunctionInfo &
 arrangeLLVMFunctionInfo(CodeGenTypes &CGT, bool instanceMethod,
                         SmallVectorImpl<CanQualType> &prefix,
                         CanQual<FunctionProtoType> FTP) {
-  SmallVector<FunctionProtoType::ExtParameterInfo, 16> paramInfos;
+  SmallExtParameterInfoList paramInfos;
   RequiredArgs Required = RequiredArgs::forPrototypePlus(FTP, prefix.size());
-  // FIXME: Kill copy.
```

Comment:
What copy is/was this referring to?

Reply:
I think that logic checks out, thanks for doing the digging to see why.  That commit is frightening... good grief.  I guess we've hit the "nothing as permanent as a temporary solution here", so ditching the comments is probably fine.  Though perhaps worth a bug-report to analyze uses of copies here to see what we can do to remove them (that we can then ignore another 16 years).